docs/tutorial/managing-resource-usage.md

```diff
@@ -1,31 +0,0 @@
-# Managing resource usage
-
-The charm can be hardware intensive, as each instance of self-hosted runner application is hosted in a virtual machine.
-
-## Virtual machine resource usage
-
-The minimum requirements for a single virtual machine are:
-
-- 1 [virtual machine vCPU](https://charmhub.io/github-runner/configure#vm-cpu)
-- 2GiB of [virtual machine memory](https://charmhub.io/github-runner/configure#vm-memory)
-- 10GiB of [virtual machine disk](https://charmhub.io/github-runner/configure#vm-disk)
-
-## Juju machine resource usage
-
-It is recommended the Juju machine has a minimum of 4GiB of memory dedicated to itself. Generally, 20GiB of disk is provisioned for the Juju machine for the Juju logs.
```

Comment:
Can't we keep this? (adapt the config option links, though)

Reply:
Mmmm but now it it totally different. The only requirement that I see could make sense is for the logs. Besides, I am not sure why we should recomment how big the github runners should be (I have run them locally with very limited resources)

Reply:
Added here to not forget: https://warthogs.atlassian.net/browse/ISD-2856